CHANGELOG.md

```diff
@@ -0,0 +1,49 @@
+# Changelog
+
+Due to the rapidly changing nature of ETH2 testnets and rapid rate of improvements to Teku, 
+we recommend most users use the latest `master` branch of Teku.
+
+## Upcoming Breaking Changes
+
+- In a future release the `--eth1-enabled` option will be removed. 
+  ETH1 will be enabled when an `--eth1-endpoint` is provided and otherwise disabled.
+
+## 0.11.2
+
+### Additions and Improvements
+
+- Updated to eth2 spec version v0.11.3.
```

Comment:
nit: is it ETH2 or eth2

Reply:
Gave up and just called it "spec".  Technically it's the beacon-chain, fork-choice, honest-validator and p2p-interface specs.